Test with Python 3.6 and Django 2.0

atodorov opened the pull request.

Natim commented on .travis.yml:
```diff
@@ -4,6 +4,7 @@ addons:
   apt:
     packages:
       - libenchant-dev
+      - python3.6
```
I wish we could the travis-ci matrix feature. See https://github.com/Kinto/kinto/blob/master/.travis.yml#L29-L33

atodorov commented:
@Natim I've made a few more updates meanwhile. Django officially advises projects to drop support for anything older than 1.11.

I have updated the changelog, added a deprecation warning if we're running on version < 2.0 and simplified the testing matrix.

Natim commented:
> Django officially advises projects to drop support for anything older than 1.11

Do you have a link for that somewhere?

Natim commented on tinymce/views.py:
```diff
@@ -4,6 +4,11 @@
+import django
+if django.VERSION < (2,):
+    import warnings
+    warnings.warn("Update to Django 2.0", DeprecationWarning)
```
We should change the message to something like: `Support for Django < 2.0 will be removed soon, please upgrade your project to use Django 2.0`

Natim commented on tox.ini:
```diff
@@ -1,16 +1,15 @@
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django110: Django>=1.10,<1.11
     django111: Django>=1.11a,<2.0
```
I think we should remove the a there.

atodorov commented:
> Do you have a link for that somewhere?

https://docs.djangoproject.com/en/2.0/releases/2.0/#third-party-library-support-for-older-version-of-django

claudep commented on tinymce/views.py:
```diff
@@ -4,6 +4,12 @@
+if django.VERSION < (2,):
+    import warnings
+    warnings.warn("Support for Django < 2.0 will be removed soon,"
+                  "please upgrade your projects to use Django 2.0", DeprecationWarning)
```
Support by whom? Django 1.11 will be supported until 2020.
As for me, I think third-party packages should not push too far for Django updates, as far as it doesn't complicate too much its code.

Natim commented:
@atodorov do you think you could add  # noqa for your deprecation warning branch?

atodorov commented:
> do you think you could add # noqa for your deprecation warning branch

@Natim can you be more precise? Do you want a noqa for the `import warnings` statement or something else 'cause I don't see flake8 complaining on my end ?

Natim commented:
You are right it is not `#noqa` it is `# pragma: nocover` for the coverage report.

atodorov commented:
@Natim what do you think about the latest changes ?

Natim commented on tinymce/views.py:
```diff
@@ -4,6 +4,12 @@
+    import warnings
+    warnings.warn("Support for Django < 2.0 will be removed soon,"
+                  "please upgrade your projects to use Django 2.0", DeprecationWarning)
+
```
I guess we don't need https://github.com/aljosa/django-tinymce/pull/224/files#diff-26390c0f125e430f2d444a575e6004baR18 anymore

atodorov replied:
You mean this piece of code:
```
 try:
     from django.urls import reverse
 except ImportError:
     # Django < 1.10
from django.core.urlresolvers import reverse
```


I think we still need it because we've not yet dropped support for Django 1.11 or anything before that. That should be removed once the warning above becomes a `raise DeprecationWarning`.

atodorov replied:
@Natim ping, please see last comment and let me know if there's still something left to do for this PR.

claudep replied:
I still don't see the point in raising a warning for the LTS version, but well...

Natim replied:
Since we don't break compatibility yet it is better to warn user that support will stop eventually.
With the slow pace updates of the lib I feel like now is a good time. Plus starting to upgrade to Django 2,0 doesn't seems like a bad idea for project to start considering.

Natim commented:
Thanks for your work on this @atodorov :)
